index.html

```diff
@@ -42,24 +42,24 @@ <h1>Welcome to Liam.L's CSD120 Website!</h1>
             <table>
                 <caption>Table 1. Current Courses</caption>
                 <thead>
-                    <tr><th rowspan = "2">Term</th><th colspan = "3">Course</th></tr>
-                    <tr><th>Code</th><th>Name</th><th>Hrs/Week</th></tr>
+                    <tr><th rowspan="2" scope="col">Term</th><th colspan="3" scope="colgroup">Course</th></tr>
+                    <tr><th scope="col">Code</th><th scope="col">Name</th><th scope="col">Hrs/Week</th></tr>
                     </thead>
                     <tbody>
-                    <tr><th rowspan = "6">Fall</th><td>CSD105</td><td>Python</td><td>3</td></tr>
+                    <tr><th rowspan="6" scope="rowgroup">Fall</th><td>CSD105</td><td>Python</td><td>3</td></tr>
                     <tr><td>CMM115</td><td>Communications</td><td>3</td></tr>
                     <tr><td>CSD120</td><td>Introduction to Web Development</td><td>4</td></tr>
                     <tr><td>CS0104</td><td>Introduction to OS and Lan</td><td>4</td></tr>
                     <tr><td>MTH122</td><td>Computer Mathematics</td><td>3</td></tr>
                     <tr><td>TNY130</td><td>Technology in Society</td><td>2</td></tr>
-                    <tr><th rowspan = "5">Winter</th><td>CSA103</td><td>Business Application</td><td>4</td></tr>
+                    <tr><th rowspan="5" scope="rowgroup">Winter</th><td>CSA103</td><td>Business Application</td><td>4</td></tr>
                     <tr><td>CSD102</td><td>Programming Using C++</td><td>5</td></tr>
                     <tr><td>CSD212</td><td>Web Scripting Languages</td><td>4</td></tr>
                     <tr><td>CSD102</td><td>Introduction to Linux</td><td>4</td></tr>
                     <tr><td>CST104</td><td>PC Hardware and Networking</td><td>5</td></tr>
                     </tbody>
                     <tfoot>
-                    <tr><th colspan = "3">Total</th><td>41</td></tr>
+                    <tr><th colspan="3" scope="row">Total</th><td>41</td></tr>
```

Comment:
Missing scope attributes
Convention is to have no spaces around attribute equal signs